examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -65,7 +64,7 @@
 # Run free orientation (vector) beamformer. Source orientation can be
 # restricted by setting pick_ori to 'max-power' (or 'normal' but only when
 # using a surface-based source space)
-stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.01, pick_ori=None)
```

Comment:
I prefer to explicit parameters to it's obvious to user that there is something hard coded here. Given the impact it can have on beamformers we should really show this.